### PR TITLE
Add subdomainless checks to diagnosis page

### LINKED ISSUE #5186: Add subdomainless checks to diagnosis page
Subdomainless URL support configuration is tricky to check, adding it to the diagnosis page would help support.

### PR DESCRIPTION
**Pull request by Kartones**
Closes #5186

### CODE REVIEW

**Comment by Kartones** [comment]
@juanignaciosl  CR please.

Tiler and SQL urls are already present so only needed to add editor and editor api ones

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -66,9 +69,15 @@ def app_diagnosis
   private
 
   def configuration_diagnosis
+    # favor displaying an organization user if any present
+    organization = Carto::Organization.first
+    user = organization.nil? ? Carto::User.first : Carto::User.where(organization_id: organization.id).first
```
Comment (juanignaciosl) [inline app/controllers/home_controller.rb]: I would pick owner instead of random user, but it's not important.

**Comment by juanignaciosl** [comment]
_minor_ suggestion. :+1:

File: app/controllers/home_controller.rb (hunk 1 above)
Comment (Kartones) [inline app/controllers/home_controller.rb]: uhm, ok, will change :D

**Comment by Kartones** [comment]
merging as I only changed the diagnosis page and tested locally and has no tests :balloon:

**Pull request merged**